devices/pine64-pinephone/u-boot/default.nix

```diff
@@ -58,29 +58,13 @@ in
     # https://patchwork.ozlabs.org/patch/1202024
```

Comment:
Move `extraPatches` to the `overrideAttrs` call like in #430 so that future Nixpkgs upgrades won't regress the package.

We can't guarantee working against a moving target.